.github/workflows/main.yml

```diff
@@ -0,0 +1,24 @@
+name: Main
+
+on: [ push, pull_request ]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup JDK 11 for Gradle
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11.0.12
+
+      - name: Clean Project
+        run: ./gradlew clean
+
+      - name: Check Code Style
+        run: ./gradlew checkCodeStyle
+
+      - name: Unit Test
+        run: ./gradlew test
```

Comment:
if build has been included in `test` command, will it be better to name it `Build and Test` like [swift](https://github.com/logto-io/swift/blob/adfa31a55b556cd795fbefc0bf102f31b3706ec9/.github/workflows/main.yml#L26)?